src/FakeItEasy.Analyzer.Shared/ArgumentConstraintAnalyzerBase.cs

```diff
@@ -21,6 +21,8 @@ public override void Initialize(AnalysisContext context)
                 throw new ArgumentNullException(nameof(context));
             }
 
+            context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.Analyze | GeneratedCodeAnalysisFlags.ReportDiagnostics);
+            context.EnableConcurrentExecution();

```

Comment:
Suggested by the "meta-analyzer"!

Reply:
What is the meta-analyzer?

Reply:
Sorry, I mean the "analyzer for writing analyzers". https://github.com/dotnet/roslyn-analyzers#microsoftcodeanalysisanalyzers